components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -839,6 +859,55 @@
     },
 
     // ---------------------------------------------------------------------
+    // Generate consents section UI
+    // ---------------------------------------------------------------------
+    _populateConsentsList: function() {
+      var _this = this;
+      if (!this._consentsModule) {
+        this._consentsModule = PhenoTips.widgets.ConsentsModule.createNew();
+      }
+      var consentsConfiguredOnRemote = true;
+      //AJAX 4: Consents present on the remote server
+      new Ajax.Request(this._serviceURL, {
+        method: _this._container.method,
+        parameters: { "do": "getremotepatientstate", 'serverid': this._selectedServer, "patientid": _this._patientId},
+          onCreate: function ()
+          {
+            console.log("PUSH: get patient state JSON - start");
```

Comment:
This is also something present throughout the code. Just matching the style that was there before. I agree though, it's utterly useless.